### PR TITLE
Add snmp test

### PR DESCRIPTION
**Pull request by asdil12**
Ticket: https://progress.opensuse.org/issues/99633
http://kazhua.qa.suse.de/tests/52

### CODE REVIEW

**Comment by foursixnine** [comment]
@asdil12 could you add a test for the access control stuff? (`community`?) basically checking that certain certain clients cannot access a resource within the network? (say expect an access denied if the client request doesn't come from a different ip that the host has.

**Comment by asdil12** [comment]
All I could think of adding would be:

```
$ snmpget -v2c -c badcommunity localhost sysLocation.0
Timeout: No Response from localhost.
```

and

```
$ snmpset -v3 -u bootstrap -l authPriv -a MD5 -x DES -A wrong_temp_password -X temp_password localhost sysLocation.0 s Earth
snmpset: Authentication failure (incorrect password, community or key) (Sub-id not found: (top) -> sysLocation)
```

**Comment by asdil12** [comment]
I added the two checks that test using an invalid community string for auth (in version 2c) and an invalid password (in version 3).

File: tests/network/snmp.pm [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Package: net-snmp
+# Summary: Test smtp server and client tools.
+# Maintainer: Dominik Heidler <dheidler@suse.de>
```
Comment (foursixnine) [inline tests/network/snmp.pm:8]: Shouldn't this be qe-core?

**Review by foursixnine — APPROVED** [review]
Aside from the Maintainer field, all looks good (And even that is not that big of a deal)

**Pull request merged**